src/helpers.coffee

```diff
@@ -233,3 +233,6 @@ exports.nameWhitespaceCharacter = (string) ->
     when '\r' then 'carriage return'
     when '\t' then 'tab'
     else string
+
+exports.normalizeStringObject = (str) ->
+  str?.valueOf?() ? str
```

Comment:
I think we can do without this function. Everywhere you’re currently calling it, you could do
```coffee
"#{str}"
```
instead. Proof:
```coffee
primitiveString = 'foo'
stringObject = new String 'foo'
primitiveString is stringObject # false
primitiveString is "#{stringObject}" # true
```